src/Styleguide/Elements/Button.tsx

```diff
@@ -176,12 +177,13 @@ export class ButtonBase extends Component<ButtonBaseProps> {
   }
 
   render() {
-    const { children, loading, ...rest } = this.props
+    const { children, loading, disabled, ...rest } = this.props
     const textProps = pick(["color", "size", "weight"], rest)
     const loadingClass = loading ? "loading" : ""
+    const disabledClass = disabled ? "disabled" : ""
 
     return (
-      <Container {...rest} className={loadingClass}>
+      <Container {...rest} className={[loadingClass, disabledClass].join(" ")}>
```

Comment:
👍

Reply:
I've typically used `classNames` lib for joining this stuff but i since classes are pretty minimal in Reaction at this point this seems fine